src/app/project/project.component.ts

```diff
@@ -1,13 +1,13 @@
-import { Component, OnInit, HostBinding } from '@angular/core';
+import { Component, OnInit, HostBinding, Inject } from '@angular/core';
```

Comment:
What does `Inject` provide here?

Reply:
Yeah, forgot to remove it after some tries...

Really miss `goimports` here 😄